### PR TITLE
First working version

### PR DESCRIPTION
**Pull request by tylerhm**
Changed LAMPAPI/Login.php to allow POST requests.

If we want to change this back to a GET request, we need to figure out [URL encoding](https://stackoverflow.com/questions/29571284/for-restful-api-can-get-method-use-json-data#:~:text=To%20answer%20your%20question,%20yes,provided%20you%20URL-encode).

Wonderful job with the API @suneettipirneni!! This has been super easy to use :)

### CODE REVIEW

**Review by suneettipirneni — APPROVED** [review]
Looks good

File: js/code.js [hunk 1]
```diff
@@ -1,65 +1,52 @@
...
+    if (login.length == 0 || password.length == 0) {
+        alert("Please provide a " + (login.length == 0 ? "username" : "password"));
+        return false;
+    }
 
+	// Hash the password
+	var hash = md5(password);
+	
+	// Create jsonPayload and api endpoint
+	var jsonPayload = '{"Login" : "' + login + '", "Password" : "' + hash + '"}';
```
Comment (suneettipirneni) [inline js/code.js]:
```suggestion
	var jsonPayload = {
	     "Login" : login,
	     "Password" : hash
	};
```

JSON in javascript can be represented like shown above, it doesn't have to be a string.

Comment (tylerhm) [reply js/code.js]: This is awesome! Will add it to the commit :)

File: LAMPAPI/Login.php [hunk 2]
```diff
@@ -12,23 +12,26 @@
 use Contactical\ErrorHandler;
 use Contactical\Error;
 
+// Removed so body is accepted
 // Ensure it's a GET Request
-ensureGET();
+// ensureGET();
```
Comment (suneettipirneni) [inline LAMPAPI/Login.php]:
```suggestion
ensurePUT();
```

This will just provide an error if we try to use a GET method again.

File: js/code.js [hunk 3]
```diff
@@ -73,75 +60,47 @@ function goToLogin()
...
-		}
-	}
-	catch(err) {
-		document.getElementById("loginStatus").innerHTML = err.message;
-	}
-		
-	jsonPayload = '{"firstName" : "' + newFirst + '", "lastName" : "' + newLast + '", "login" : "' + login + '", "password" : "' + password + '"}';
-	url = urlBase + '/CreateAccount.' + extension;
+	// Create jsonPayload and api endpoint
+	jsonPayload = '{"FirstName" : "' + newFirst + '", "LastName" : "' + newLast + '", "Login" : "' + login + '", "Password" : "' + hash + '"}';
```
Comment (suneettipirneni) [inline js/code.js]: Caught another
```suggestion
	jsonPayload = {
	     "FirstName": newFirst,
	     "LastName": newLast,
	     "Login": login,
	     "Password": hash
	};
```

**Review by suneettipirneni — CHANGES_REQUESTED** [review]

**Pull request merged**